CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.5] - 2020-11-28
```

Comment:
Since there are no changes to functionality, we won't need to submit a release with this change.

Reply:
Sounds good! I backed out all version bump changes (including the bump in `package.json`). Do we still want to bump `package.json` to v1.2.5 or is that sufficient?

Reply:
Thanks! Keeping it at v1.2.4 should be good 👍🏻